src/dotnet/CarbonAware/Plugin/Constants.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace CarbonAware.Plugin;
```

Comment:
This is in the wrong folder name.  The folder should be "PlugIns" to conform to the naming of other items.